### PR TITLE
fix: add overrides for some of the content schemas

### PR DESCRIPTION
**Pull request by wolfy1339**
This will allow to have some better discrimination between schemas

### CODE REVIEW

File: scripts/overrides/index.js [hunk 1]
```diff
@@ -67,6 +67,19 @@ function overrides(file, schema) {
           `and then update \`SUPPORTED_GHES_VERSION\`.`;
   }  
 
+
+  // Fix for the `GET /repos/{owner}/{repo}/contents/{path}` endpoint.
+  // This is so we can discriminate between the different response types.
+  // ref: https://github.com/github/rest-api-description/issues/165
+  // ref: https://github.com/octokit/rest.js/issues/32
+  const defenitions = schema.components.schemas;
```
Comment (oscard0m) [inline scripts/overrides/index.js:75]: This line is giving problems with our CI: https://github.com/octokit/openapi/runs/7258962452?check_suite_focus=true

Comment (gr2m) [reply scripts/overrides/index.js:75]: also typo, should be `definitions`

File: scripts/overrides/index.js [hunk 2]
```diff
@@ -67,6 +67,19 @@ function overrides(file, schema) {
...
+
+  defenitions['content-file'].items.properties.type.enum = ["file"];
+  defenitions['content-symlink'].items.properties.name.enum = ["symlink"];
+  defenitions['content-submodule'].title = "Submodule Content";
+  defenitions['content-submodule'].description = "An object describing a submodule.";
+  defenitions['content-submodule'].items.properties.type.enum = ["submodule"];
```
Comment (oscard0m) [inline scripts/overrides/index.js:81]:
```suggestion
  const definitions = schema.components.schemas;

  definitions['content-file'].items.properties.type.enum = ["file"];
  definitions['content-symlink'].items.properties.name.enum = ["symlink"];
  definitions['content-submodule'].title = "Submodule Content";
  definitions['content-submodule'].description = "An object describing a submodule.";
  definitions['content-submodule'].items.properties.type.enum = ["submodule"];
```

**Comment by wolfy1339** [comment]
This will only work for referenced schemas.

I'm not sure how you would proceed to fix this for the de-referenced schema

**Comment by wolfy1339** [comment]
Fixed upstream

**Pull request closed without merging**